index.js

```diff
@@ -6,13 +6,19 @@ var through = require("through2");
 // # Filter
 // Create a through stream that only passes data that passes the given test
 // function
-module.exports = function(test, opt) {
+var filter = module.exports = function(test, opt) {
 	return through(opt || {}, function(data, _, cb) {
 		if (test(data)) cb(null, data);
 		else cb(null);
 	});
 };
 
+module.exports.obj = function(test, opt) {
+	opt = opt || {}
+	opt.objectMode = true
```

Comment:
[`Object.assign` is not supported in IE11, Safari 8 and Node 0.12](http://kangax.github.io/compat-table/es6/#test-Object_static_methods_Object.assign). For this module, do you suggest using shims or shall I use `xtend`?